### PR TITLE
Update Beta User WDL to use bulk ingest [VS-982]

### PR DESCRIPTION
**Pull request by rsasch**
latest run of Beta User WDL with bulk ingest and no data table column names provided: https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/79337ea3-593f-4896-b056-9f47bfd7c7c7

### CODE REVIEW

File: .dockstore.yml [hunk 1]
```diff
@@ -119,18 +119,14 @@ workflows:
...
-     primaryDescriptorPath: /scripts/variantstore/wdl/GvsPrepareBulkImport.wdl
-     filters:
-       branches:
-         - ah_var_store
    - name: GvsBulkIngestGenomes
      subclass: WDL
      primaryDescriptorPath: /scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl
      filters:
        branches:
+         - master
```
[the remark this answers is not in the document]
Comment (rsasch) [reply .dockstore.yml:126]: I can dream, can't I?

File: scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl [hunk 2]
```diff
@@ -1,221 +1,236 @@
...
+      input:
+        workspace_id = GetWorkspaceId.workspace_id,
+        workspace_name = GetWorkspaceName.workspace_name,
+        workspace_namespace = GetWorkspaceName.workspace_namespace,
+        samples_table_name = samples_table_name,
+        sample_id_column_name = sample_id_column_name,
+    }
+  }
+
+  call GenerateFOFNsFromDataTables {
```
[the remark this answers is not in the document]
Comment (rsasch) [reply scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl:44]: Thanks?  It's from `GvsPrepareBulkImport.wdl`

File: scripts/variantstore/wdl/extract/get_columns_for_import.py [hunk 3]
```diff
@@ -243,13 +243,13 @@ def get_column_values(columnSamples, numSamples, user_defined_vcf, user_defined_
         # Check the contents of the columns: the duck algorithm.  If its contents LOOK like vcfs and indexes, go from there
         for col in path_ends_in_vcf_gz:
             # ...and has an analogue that looks like an index file?
-            index_column = f"{column}_index"
+            index_column = f"{col}_index"
```
Comment (mcovarr) [inline scripts/variantstore/wdl/extract/get_columns_for_import.py:246]: uh what was this doing before these changes?

**Review by mcovarr — APPROVED** [review]

File: scripts/variantstore/wdl/extract/get_columns_for_import.py (hunk 3 above)
Comment (rsasch) [reply scripts/variantstore/wdl/extract/get_columns_for_import.py:246]: failing!

**Comment by mcovarr** [comment]
Heads up my one-FOFN-to-rule-them-all PR ended up turning into a PR that does (or at least tries to) do [all the bulk ingest things](https://github.com/broadinstitute/gatk/pull/8404). Happy to discuss in mobbing

**Comment by rsasch** [comment]
closing in favor of https://github.com/broadinstitute/gatk/pull/8404 (and possible other future PRs)

**Pull request closed without merging**